cneuromax/projects/friends_language_encoder/litmodule_peft.py

```diff
@@ -41,18 +37,9 @@ def __init__(
         super().__init__(*args, **kwargs)
         self.config: FriendsLitModuleConfig
 
-        for param in self.nnmodule.parameters():
-            param.requires_grad = False
-
-        for layer_name in self.config.layer_names:
-            for name, param in self.nnmodule.named_parameters():
-                if layer_name in name:
-                    param.requires_grad = True
-
     def step(
         self: "FriendsFinetuningModel",
         batch: BatchEncoding,
-        stage: Any[str, one_of("train", "val", "test", "predict")],
```

Comment:
this has to stay, see https://github.com/courtois-neuromod/cneuromax/blob/main/cneuromax/fitting/deeplearning/litmodule/base.py#L50